### DISCUSSION TITLE
Vote for arkbriar as RisingLight member

### DISCUSSION THREAD

**Discussion by TennyZhuang** [risinglightdb/risinglight#521] [Member]
@arkbriar has made great contributions to the RisingLight in the past month. 

In #437, @arkbriar add support for parallelized fetching in join executors.

In #467, @arkbriar implemented a difficult feature: Cancelable query based on managed tokio tasks.

In #499, @arkbriar separated the write logic from the rowset builder and fixes the panic in #493.

He has many other contributions, and here's a list of his pull requests: https://github.com/risinglightdb/risinglight/pulls?q=author%3Aarkbriar

@arkbriar demonstrated a good understanding of the Rust programming language and the RisingLight project. Therefore, I'd like to promote him as RisingLight member.

If you are a member of the RisingLight organization, you may vote for this proposal with 👍 and 👎. According to [GOVERNANCE](https://github.com/risinglightdb/risinglight/blob/main/GOVERNANCE.md), a minimum requirement of 2 votes are required for accepting this proposal. Thanks!

**Comment by skyzh** [comment]
With 7 votes from @TennyZhuang @skyzh @st1page @wangrunji0408 @zzl200012 @Fedomn @xxchan, we are accepting this vote. Welcome to the RisingLight community! The invitation email should have been sent. And @arkbriar will be granted write access to all our repos.

**Comment by arkbriar** [comment]
Thank you. I'll keep contributing to the RisingLight community.
